### ISSUE TITLE
CSAR deployment fails with `UnicodeDecodeError`

### ISSUE THREAD

**Issue by duelle** [xlab-si/xopera-opera#78]
Version: `opera 0.5.8`

When trying to deploy the [sut.zip](https://github.com/xlab-si/xopera-opera/files/4970638/sut.zip) file (renamed to zip to be able to upload it) with `opera deploy sut.csar`, it fails with the following error message:

```
% opera deploy sut.csar
Traceback (most recent call last):
  File "/usr/bin/opera", line 8, in <module>
    sys.exit(main())
  File "/home/duelle/.local/lib/python3.8/site-packages/opera/cli.py", line 41, in main
    return args.func(args)
  File "/home/duelle/.local/lib/python3.8/site-packages/opera/commands/deploy.py", line 76, in deploy
    ast = tosca.load(Path.cwd(), PurePath(service_template))
  File "/home/duelle/.local/lib/python3.8/site-packages/opera/parser/tosca/__init__.py", line 16, in load
    input_yaml = yaml.load(input_fd, str(template_name))
  File "/home/duelle/.local/lib/python3.8/site-packages/opera/parser/yaml/__init__.py", line 7, in load
    return loader.get_single_data()
  File "/usr/lib/python3.8/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "ext/_yaml.pyx", line 702, in _yaml.CParser.get_single_node
  File "ext/_yaml.pyx", line 903, in _yaml.CParser._parse_next_event
  File "ext/_yaml.pyx", line 912, in _yaml.input_handler
  File "/usr/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdc in position 10: invalid continuation byte
```

**Comment by duelle** [comment]
Same for [http.zip](https://github.com/xlab-si/xopera-opera/files/4970654/http.zip)

**Comment by anzoman** [comment]
Hi @duelle,

thanks for bringing this up. The thing is that the latest version of opera (which is now **0.5.8**) supports the compressed version of CSAR but in this case deployment process includes one additional step - you have to initialize the CSAR file (this was introduced in #55). This is done with the new `opera init` that takes a template file or a zipped CSAR file (and an optional YAML file with inputs). When the compressed CSAR is provided to the opera init command it is then validated to be sure that the CSAR is compliant with TOSCA CSAR guidelines.

So basically what you have to do is the following:

```console
# initialize the CSAR (from the prepared compressed CSAR file) 
opera init sut.zip

# deploy the previously initialized CSAR
opera deploy
```

**Comment by anzoman** [comment]
This issue can be closed since we resolved the problems with the usage of right opera commands to initialize and deploy a (compressed) CSAR.
